0015-estimator-interface.md

```diff
@@ -22,7 +24,7 @@ It is common, if not typical, for a user to want to estimate many observables co
 Likewise, it is common, if not typical, for a user to want to supply multiple parameter value sets for the same circuit.
 This RFC proposes to fix these issues by changing the `Estimator.run()` signature in the following ways: 
 
- 1. Take the transpose of the current signature; rather than accepting `circuits`, `parameter_values`, and `observables` as three different (iterable) arguments which are to be zipped together, instead localize the distinct tasks to be run via an iterable of triples `(circuit, parameter_values, observables)`.
+ 1. Take the transpose of the current signature; rather than accepting `circuits`, `parameter_values`, and `observables` as three different (iterable) arguments which are to be zipped together, instead localize the distinct PUBs to be run via an iterable of triples `(circuit, parameter_values, observables)`.
```

Comment:
PUBs have not been mentioned yet so we can just avoid saying it, since this sentence is using it in a colloquial way.

```suggestion
 1. Take the transpose of the current signature; rather than accepting `circuits`, `parameter_values`, and `observables` as three different (iterable) arguments which are to be zipped together, instead localize them into an iterable of triples `(circuit, parameter_values, observables)`.
```